### ISSUE TITLE
Visual feedback for touchpad clicks

### ISSUE THREAD

**Issue by InitialDet** [Lime3DS/Lime-3DS-Emulator#383]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What feature are you suggesting?

A visual feedback of the location clicked in the touchpad screen.

### Why would this feature be useful?

When using the PS4 touchpad, I noticed that I often missed some menu clicks. I couldn't tell if I was tapping too far from or very close to the button. It would be really helpful to have a visual feedback, like Android does, so we know exactly where we clicked and don't need to keep randomly tapping different spots on the touchpad until it works.
![img](https://i.imgur.com/SztUYS8.gif)

**Comment by DavidRGriswold** [comment]
So this would basically be a little circle that appears around the cursor (if using mouse mode) or appear then disappears like 250ms later if using something like a touchscreen or trackpad?

**Comment by InitialDet** [comment]
> So this would basically be a little circle that appears around the cursor (if using mouse mode) or appear then disappears like 250ms later if using something like a touchscreen or trackpad?

Yeah. Basically any effect, or circle like in the gif i used as an example, to show where the click happened. Not sure it would be useful when using a mouse, but for the trackpad it would be extremely helpful.
